src/app/layout/navBar.scss

```diff
@@ -1,84 +1,101 @@
 @import "./color.scss";
-//Common design for button
+
+// Font sizes
+$font-size-small: 14px;
+$font-size-medium: 16px;
+$font-size-large: 18px;
+
+// Common design for button
 @mixin button-design{
     color: $white;
-    padding: 15px;
+    padding: $font-size-medium;
     background-color: $green;
     border-radius: 5px;
     border: none;
     cursor: pointer;
 }
-//Header container design
+
+// Header container design
 .nav-header {
     display: flex;
     width: 100%;
-    height: 80px; 
+    height: 80px;
     justify-content: space-between;
     border: none;
     background-color: $light-gray;
     overflow-x: hidden;
-    //Image design
+
+    // Image design
     .nav-img {
-        margin: 10px; 
-        height: 60px; 
+        margin: $font-size-small;
+        height: 60px;
     }
+
     // Login and signup design
     .log-sign {
         display: flex;
-        gap: 20px; 
+        gap: $font-size-medium;
         align-items: center;
-        //Login design
+
+        // Login design
         .nav-log {
             color: $white;
+
             button {
                 @include button-design();
+                font-size: $font-size-small;
             }
-            
-            button:hover{
+
+            button:hover {
                 background-color: $light-gray;
                 color: $green;
             }
         }
-        //Signup design
+
+        // Signup design
         .nav-signUP {
             color: $white;
+
             button {
-                margin-right: 15px;
+                margin-right: $font-size-medium;
                 @include button-design();
+                font-size: $font-size-small;
             }
-            
-            button:hover{
+
+            button:hover {
                 background-color: $light-gray;
                 color: $green;
             }
         }
     }
 }
-//For tablet mode
+
+// For tablet mode
 @media (max-width: 767px) {
     .nav-header {
-        height: 60px; 
-        
+        height: 60px;
+
         .nav-img {
-            margin-left: 20px; 
-            height: 45px; 
+            margin-left: $font-size-medium;
```

Comment:
don't use $font-size-medium variable for margin, in case you decided to change the font size then margin will also get affected so either create one or can write direct margin value